eng/pipelines/runtime.yml

```diff
@@ -598,6 +602,7 @@ jobs:
           eq(dependencies.checkout.outputs['SetPathVars_libraries.containsChange'], true),
           eq(dependencies.checkout.outputs['SetPathVars_coreclr.containsChange'], true),
           eq(dependencies.checkout.outputs['SetPathVars_mono.containsChange'], true),
+          eq(dependencies.checkout.outputs['SetPathVars_runtimetests.containsChange'], true),
```

Comment:
Just double checking to refresh my memory, libraries tests are needed for runtime tests, right?

Reply:
I don't think we need any library test build artifacts for building or running runtime tests, just the libraries build. I can easily remove this line if you believe it's superfluous, I guess I'll probably catch a few more conflicts before the night is over, I just hope that around PST midnight the churn will calm down a bit. I'm not going to update the PR right away as that would cancel the running tests and that would be a waste of lab resources considering this is ultimately just a "perf optimization".

Reply:
I think we can update this as a subsequent PR to not risk getting more merge conflicts.